pb/block.proto

```diff
@@ -7,20 +7,9 @@ import "pb/header.proto";
 import "pb/transaction.proto";
 
 
-//option (gogoproto.gostring_all) = true;
-//option (gogoproto.goproto_stringer_all) = false;
-//option (gogoproto.stringer_all) =  true;
-//option (gogoproto.marshaler_all) = true;
-//option (gogoproto.sizer_all) = true;
-//option (gogoproto.unmarshaler_all) = true;
-//
-//option (gogoproto.testgen_all) = true;
-//option (gogoproto.equal_all) = true;
-//option (gogoproto.populate_all) = true;
-
 
 message MsgBlock {
-	BlockHeader     Header                 = 1;
-    repeated MsgTx  Transactions           = 2;
+	BlockHeader     header                 = 1;
+    repeated MsgTx  transactions           = 2;
```

Comment:
Also formatting issue here

Reply:
done

Reply:
Suggest run go format before commit each time

Reply:
this is .proto file. not a .go file...